src/main/java/com/dumbdogdiner/stickyapi/common/util/MathUtil.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright (c) 2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util;
+
+import org.bukkit.Location;
+import org.bukkit.util.Vector;
+
+import java.text.CharacterIterator;
+import java.text.StringCharacterIterator;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * <p>
+ * Util class for commonly used math operations.
+ * </p>
+ */
+public class MathUtil {
+
+    private static final Random random = new Random();
+
+    /**
+     * Get a random number within a range
+     *
+     * @param max maximum value
+     * @return a random integer within the specified range
+     */
+    public static int rInt(int max) {
+        return random.nextInt(max);
+    }
+
+    /**
+     * Get a random number within a range
+     *
+     * @param min minimum value
+     * @param max maximum value
+     * @return a random integer within the specified range
+     * @throws IllegalArgumentException when min is greater than max
+     */
+    public static int rInt(int min, int max) {
+        if (min >= max)
+            throw new IllegalArgumentException("Min may not be greater than max!");
+        return min + rInt(1 + max - min);
+    }
+
+    /**
+     * Get a random number within a range
+     *
+     * @param max maximum value
+     * @return a random double within the specified range
+     */
+    public static double rDouble(double max) {
+        return max * random.nextDouble();
+    }
+
+    /**
+     * Get a random number within a range
+     *
+     * @param min minimum value
+     * @param max maximum value
+     * @return a random double within the specified range
+     * @throws IllegalArgumentException when min is greater than max
+     */
+    public static double rDouble(double min, double max) {
+        if (min >= max)
+            throw new IllegalArgumentException("Min may not be greater than max!");
+        return min + (max - min) * random.nextDouble();
+    }
+
+    /**
+     * Get the offset between two {@link org.bukkit.Location} objects
+     * in a 3d environment
+     *
+     * @param pointA first location
+     * @param pointB second location
+     * @return the offset (distance) between the given locations
+     */
+    public static double offset(Location pointA, Location pointB) {
+        return offset(pointA.toVector(), pointB.toVector());
+    }
+
+    /**
```

Comment:
Okay i'm tired of making these suggestion comments, please fix your javadocs according to our [guidelines](https://google.github.io/styleguide/javaguide.html#s7-javadoc)